sourcemash/frontend/static/js/routers/app_router.js

```diff
@@ -6,6 +6,7 @@ Sourcemash.Routers.AppRouter = Backbone.Router.extend({
         "categories/:id": "showCategory",
         "saved": "showSaved",
         "browse": "browseFeeds",
+        "categorizer": "showCategorizer",
```

Comment:
You find the functionaly by using sourcemash! The ability to type in a url and see categories is useless to everyone except the judges.

-- 
Alex Gerstein
gerstein.alex@gmail.com

On May 24, 2015 at 10:40:21 PM, Scott Gladstone (notifications@github.com) wrote:

In sourcemash/frontend/static/js/routers/app_router.js:

> @@ -6,6 +6,7 @@ Sourcemash.Routers.AppRouter = Backbone.Router.extend({
>          "categories/:id": "showCategory",
>          "saved": "showSaved",
>          "browse": "browseFeeds",
> +        "categorizer": "showCategorizer",
It seems outrageous to me that we offer this functionality (and plan to pitch it as the fundamental core of our product), yet also make it impossible for anybody to find
—
Reply to this email directly or view it on GitHub.